java-sdk-example/src/test/java/CreateSandboxTestWithCustomPatch.java

```diff
@@ -0,0 +1,136 @@
+import com.signadot.ApiClient;
+import com.signadot.ApiException;
+import com.signadot.api.SandboxesApi;
+import com.signadot.model.*;
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+public class CreateSandboxTestWithCustomPatch {
+
+  public static final String HOTROD = "hotrod";
+  public static final String CUSTOM_PATCH = "spec:\n" +
+                                            "  template:\n" +
+                                            "    spec:\n" +
+                                            "      containers:\n" +
+                                            "      - name: hotrod\n" +
+                                            "        env:\n" +
+                                            "        - name:  PATCH_TEST_VAR\n" +
+                                            "          value: foo\n";
+
+  public static final String ORG_NAME = System.getenv("SIGNADOT_ORG");
+  public static final String SIGNADOT_API_KEY = System.getenv("SIGNADOT_API_KEY");
+  public static final String CLUSTER_NAME = System.getenv("SIGNADOT_CLUSTER_NAME");
+
+  private static RequestSpecification requestSpec;
+
+  ApiClient apiClient;
+  SandboxesApi sandboxesApi;
+  CreateSandboxResponse response;
+  String sandboxID;
+
+  @BeforeSuite
+  public void createSandboxWithCustomPatch() throws ApiException, InterruptedException {
+    apiClient = new ApiClient();
+    apiClient.setApiKey(SIGNADOT_API_KEY);
+    sandboxesApi = new SandboxesApi(apiClient);
+
+    String sandboxName = String.format("custom-patch-test-%s", RandomStringUtils.randomAlphanumeric(5));
+
+    SandboxFork frontendFork = new SandboxFork()
+      .forkOf(new ForkOf().kind("Deployment").namespace(HOTROD).name("frontend"))
+      .customizations(new SandboxCustomizations()
+        .addImagesItem(new Image().image("signadot/hotrod:49aa0813feba0fb74e4edccdde27702605de07e0"))
```

Comment:
Let's assign the image tag as a variable as it's being used for the other fork too.